sdks/python/apache_beam/runners/portability/stager.py

```diff
@@ -774,7 +774,7 @@ def _build_setup_package(setup_file,  # type: str
       if build_setup_args is None:
         # if build is installed in the user env, use it to
         # build the sdist else fallback to legacy setup.py sdist call.
-        if importlib.util.find_spec('build'):
+        if pkgutil.find_loader('build') is not None:
```

Comment:
this is a deprecated method that will be removed in a future version of Python.

Reply:
Using try except instead.